README.md

```diff
@@ -34,18 +34,6 @@ Move into the newly created directory:
 ```
 cd mbed-os-example-sd-driver
 ```
-	
-If the mbed-os library was not automatically added (see trace above), do the following to import mbed-os:
-
-```
-mbed new .
-```
```

Comment:
I think this happens (if the user uses an up-to-date Mbed CLI). Other examples don't mention it at all.

Reply:
It's should be fine now that the library is part of mbed-os